### ISSUE TITLE
incorrect restoration of an existing part table

### ISSUE THREAD

**Issue by dimitri-yatsenko** [datajoint/datajoint-matlab#80]
If a part table already exists, then `dj.new` will incorrectly reconstruct the `classdef file` as a master table.

**Comment by eywalker** [comment]
So this means that `dj.new` has to correctly inspect the table name to detect that it's a part table and create corresponding class with the correct inheritance, right?

**Comment by dimitri-yatsenko** [comment]
Yes. In general, `dj.new` should be approved for part tables to suggest the correct naming for the given master.
